ci/builder/requirements.txt

```diff
@@ -4,3 +4,5 @@ PyMySQL==1.0.2
 pyyaml==6.0
 semver==3.0.0-dev.2
 toml==0.10.2
+scipy==1.7.3
```

Comment:
@benesch is this the right place those dependencies?

Reply:
Eep, this is tricky. We want to avoid heavyweight slash hard to install deps here because every user who wants to run a demo has to install them. For now I'd say put them in requirements-dev.txt and just document in the readme for the feature bench directory that you need to run `bin/pyactivate --dev` once to get the correct dependencies installed. We can do something more clever in the future.

Reply:
Thanks, will do!